keps/sig-apps/2232-suspend-jobs/README.md

```diff
@@ -385,78 +389,84 @@ field.
 
-* **How can a rollout fail? Can it impact already running workloads?**
-  Try to be as paranoid as possible - e.g., what if some components will restart
-   mid-rollout?
-
-* **What specific metrics should inform a rollback?**
-
-* **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
+* **How can a rollout fail? Can it impact already running workloads?** Impact
+  to existing Jobs that previously didn't use this feature in alpha is
+  impossible. In workloads using the feature in an older version, suspended
+  Jobs may inadvertently be resumed (or Jobs may be inadvertently suspended) if
+  there are storage-related issues arising from components crashing
+  mid-rollout.
+
+* **What specific metrics should inform a rollback?** `job_sync_duration_seconds`
+  should be observed. Unexpected spikes in the metric with label
+  `action=pods_deleted` is potentially an indicator that:
+    1. Jobs are getting suspended when they shouldn't be.
+    2. Job sync latency is high when Job are suspended.
+  While the above list isn't exhaustive, they're signals in favour of rollbacks.
+
+* **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path
+  tested?** <!-- I'll answer this after implementation.
   Describe manual testing that was done and the outcomes.
   Longer term, we may want to require automated upgrade/rollback tests, but we
-  are missing a bunch of machinery and tooling and can't do that now.
+  are missing a bunch of machinery and tooling and can't do that now. -->
 
-* **Is the rollout accompanied by any deprecations and/or removals of features, APIs, 
-fields of API types, flags, etc.?**
-  Even if applying deprecation policies, they may still surprise some users.
+* **Is the rollout accompanied by any deprecations and/or removals of features,
+  APIs, fields of API types, flags, etc.?** No.
 
 ### Monitoring Requirements
 
 _This section must be completed when targeting beta graduation to a release._
 
 * **How can an operator determine if the feature is in use by workloads?**
-  Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
-  checking if there are objects with field X set) may be a last resort. Avoid
-  logs or events for this purpose.
-
-* **What are the SLIs (Service Level Indicators) an operator can use to determine 
-the health of the service?**
-  - [ ] Metrics
-    - Metric name:
-    - [Optional] Aggregation method:
+  The `.spec.suspend` field is set to true by Jobs. The status conditions of a
+  Job can also be used to determine whether a Job is using the feature (look
+  for a condition of type "Suspended").
+
+* **What are the SLIs (Service Level Indicators) an operator can use to
+  determine the health of the service?**
+  - [x] Metrics
+    - Metric name: `job_sync_duration_seconds` gets a new label named `action`
+      to allow operators to filter Job sync latency by the action performed.
+      The possible values for this label are:
```

Comment:
I think we should add the same label to `job_sync_total`. That would also help determine if errors are related to specific actions.